github/ci/services/prometheus-stack/manifests/service-monitors/common/servicemonitors.yaml

```diff
@@ -48,31 +48,31 @@ spec:
   endpoints:
   - port: http-metrics
     bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
----
-# Source: kube-prometheus-stack/templates/exporters/kube-controller-manager/servicemonitor.yaml
-apiVersion: monitoring.coreos.com/v1
-kind: ServiceMonitor
-metadata:
-  name: prometheus-stack-kube-prom-kube-controller-manager
-  namespace: default
-  labels:
-    app: kube-prometheus-stack-kube-controller-manager
-    chart: kube-prometheus-stack-13.6.0
-    release: "prometheus-stack"
-    heritage: "Helm"
-    group: kubevirtci
-spec:
-  jobLabel: jobLabel
-  selector:
-    matchLabels:
-      app: kube-prometheus-stack-kube-controller-manager
-      release: "prometheus-stack"
-  namespaceSelector:
-    matchNames:
-      - kube-system
-  endpoints:
-  - port: http-metrics
-    bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+# ---
```

Comment:
I think it would be better to remove the service monitors not being used instead of keeping dead code, wdyt?

Reply:
Removed! Can you approve and lgtm?